BaseFilter/sections/specific.txt

```diff
@@ -180,6 +180,11 @@ youtubekids.com,youtube-nocookie.com,youtube.com#%#//scriptlet('set-constant', '
 /youtubei/v*/player?key=$replace=/\"playerAds.*?\}\}\]\,//,domain=youtubekids.com|youtube-nocookie.com|youtube.com
 !#endif
 !#if (!adguard_app_windows && !adguard_app_mac && !adguard_app_android && !adguard_ext_firefox)
+! https://github.com/AdguardTeam/AdguardFilters/pull/170936
```

Comment:
Cannot check where they should be applied, but it seems nothing is broken (checked on desktop, android and ios)

Reply:
This PR is just a scriptlet version of https://github.com/AdguardTeam/AdguardFilters/pull/170936 to cover Chrome and Safari extension.

Reply:
@AdamWr @piquark6046 Please also take a look.

Reply:
After quick test I don't see any breakage on my end with these rules.

By the way, if I'm not wrong, rule like `~tv.youtube.com,youtube.com#%#` doesn't work in Safari on both websites.

Reply:
yep

Reply:
Can I change to `www.youtube.com` then? https://github.com/AdguardTeam/AdguardForiOS/issues/1897 is closed.

Reply:
Will check on iOS.
Is/was something broken on accounts.youtube.com, m.youtube.com, payments.youtube.com and tv.youtube.com?

Reply:
m.youtube.com, music.youtube.com works fine

Reply:
> Is/was something broken 

No, but as the anti-adb is shown only on www it'll be best to inject only on www.